gradle/bintray.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.jfrog.bintray'
```

Comment:
This file is already in the shared build config, so you should be able to use `../config/publish/bintray.gradle` instead.

Reply:
Thanks for sharing. I will update it out and rebase.